src/test/fixtures/config/forms/local/cover-letter-extension.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "Covering Letter (extension)",
+    "type": "VERIFIABLE_DOCUMENT",
+    "defaults": {
+      "$template": {
+        "type": "EMBEDDED_RENDERER",
+        "name": "COVERING_LETTER",
+        "url": "https://generic-templates.tradetrust.io"
+      },
+      "issuers": []
+    },
+    "schema": {
+      "type": "object",
+      "properties": {
+        "logo": {
+          "type": "string",
+          "title": "Document Title"
+        },
+        "title": {
+          "type": "string",
+          "title": "Document Title"
+        },
+        "remarks": {
+          "type": "string",
+          "title": "Remarks"
+        },
+        "backgroundColor": {
+          "type": "string",
+          "title": "Background Color"
+        },
+        "titleColor": {
+          "type": "string",
+          "title": "Title Color"
+        },
+        "remarksColor": {
+          "type": "string",
+          "title": "Remarks Color"
+        }
+      }
+    },
+    "uiSchema": {
+      "logo": {
+        "ui:widget": "file",
+        "ui:options": {
+          "text": "Upload Cover Letter Logo",
+          "accept": ".png, .jpeg, .jpg"
+        }
+      },
+      "remarks": {
+        "ui:widget": "textarea"
+      },
+      "backgroundColor": {
+        "ui:widget": "color",
+        "ui:options": {
+          "emptyValue": "#ffffff"
+        }
+      },
+      "titleColor": {
+        "ui:widget": "color",
+        "ui:options": {
+          "emptyValue": "#4e4e50"
+        }
+      },
+      "remarksColor": {
+        "ui:widget": "color",
+        "ui:options": {
+          "emptyValue": "#4e4e50"
+        }
+      }
+    },
+    "attachments": {
+      "allow": true
+    },
+    "extension": "docTest",
+    "headers": ["title", "remarks"]
```

Comment:
i see headers are here, update tradetrust documentation on what's the use and how to use?

Reply:
i will remove the mention of headers. this seems to be removed after the implementation of "download .csv file"

Reply:
um so there is no longer a use for headers? what was it for

Reply:
previously is it used to deteremine the header of the data csv files

Reply:
yup no longer a use for headers